lib/ansible/plugins/connection/winrm.py

```diff
@@ -359,7 +359,7 @@ def put_file(self, in_path, out_path):
         if result.status_code != 0:
             raise AnsibleError(to_native(result.std_err))
 
-        put_output = json.loads(result.std_out)
+        put_output = json.loads(to_text(result.std_out))
```

Comment:
In what case is the JSON output encoded in an encoding other than UTF-8 here?  `to_text()` looks fine to me.